mobile/src/components/interview/components/applications/components/application-form/styles.ts

```diff
@@ -54,6 +54,12 @@ const styles = StyleSheet.create({
   buttonHide: {
     opacity: 0,
   },
+  datePickerWrapper: {
+    minWidth: '100%',
+    borderRadius: 8,
+    borderWidth: 2,
+    borderColor: AppColor.BACKGROUND.GRAY_400,
```

Comment:
Should have same look as input
```suggestion
```

Reply:
## Without these styles date picker looks like this

<img src="https://gyazo.com/ef4ff62916ffded2ea813e5a22887d38.png" width="45%"> <img src="https://gyazo.com/c2fb0ae6d7d7080c8b7d7fe04894737c.png" width="45%">

Reply:
It's ok

Reply:
ok, let's add border but let it be lighter like text color for values

Reply:
borderWidth 1

Reply:
<img src="https://gyazo.com/085c9b7a2a9b644500cd8d8de1077cb4.png" width="45%">